view/frontend/web/js/view/checkout/email.js

```diff
@@ -1,40 +1,61 @@
 define([
     'uiComponent',
     'jquery',
-    'jquery/ui'
+    'jquery/ui',
+    '!domReady'
```

Comment:
why is this needed?  I don't see anything in the added code pertaining to it

Reply:
it makes the code not load until the document is ready. https://requirejs.org/docs/api.html#pageload